src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api/api';
+import { Loader } from './Loader';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeolpe] = useState<Person[]>([]);
+  const [isLoading, setLoading] = useState(false);
+
+  useEffect(() => {
+    const loadingPeole = async () => {
+      const peopleFromServer = await getPeople();
+
+      setPeolpe(peopleFromServer);
+      setLoading(true);
```

Comment:
why do you set `loading` to `true` at the end?

Reply:
yeah, I suppose you should have done the opposite